vizro-core/src/vizro/models/_controls/filter.py

```diff
@@ -280,6 +288,7 @@ def _get_min_max(targeted_data: pd.DataFrame, current_value=None) -> tuple[float
         # Use item() to convert to convert scalar from numpy to Python type. This isn't needed during pre_build because
         # pydantic will coerce the type, but it is necessary in __call__ where we don't update model field values
         # and instead just pass straight to the Dash component.
+        # AM QUESTION: why could AttributeError be raised here?
```

Comment:
17 tests fail if I remove this, and the reason is that `targeted_data.min(axis=None)` and `targeted_data.max(axis=None)` for some inputs return a pure `int` or `Timestamp` which has no `item()` and are already in the final form we expect.

Reply:
Ah yes, I see. Let's add a comment to explain that.
```suggestion
        # In some cases _min and _max are already Python types and so item() call is not required and would raise an AttributeError.
```